<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="UT-LTP-619" failures="2" tests="2" name="Test" time="3.032" errors="0" timestamp="10 May 2021 11:25:08 GMT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.995">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.test.APIAutomation.OneDat.StepDefinitions.StepDefinition.the_api_call_is_success_with_status_code(StepDefinition.java:52)
at ✽.the Api call is success with status code 200(file:///E:/localrepo/src/test/java/com/test/APIAutomation/OneDat/FeatureFiles/bookValidation.feature:6)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.463">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.test.APIAutomation.OneDat.StepDefinitions.StepDefinition.the_api_call_is_success_with_status_code(StepDefinition.java:52)
at ✽.the Api call is success with status code 200(file:///E:/localrepo/src/test/java/com/test/APIAutomation/OneDat/FeatureFiles/bookValidation.feature:12)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Test -->
